Sources/SourceKitLSP/Swift/MacroExpansion.swift

```diff
@@ -10,6 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Crypto
```

Comment:
Could you add `Crypto` as a dependency of `SourceKitLSP` in the Package.swift? Ie. add 
```swift
.product(name: "Crypto", package: "swift-crypto"),
```

to the dependencies of the `SourceKitLSP` target?